client/src/App.jsx

```diff
@@ -1,43 +1,38 @@
 import { useState } from 'react';
-import logo from './logo.svg';
+import InstagramLogo from './instagram_icon.png';
 import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0);
-
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>Hello Vite + React!</p>
-        <p>
-          <button type="button" onClick={() => setCount((count) => count + 1)}>
-            count is: {count}
-          </button>
-        </p>
-        <p>
-          Edit <code>App.jsx</code> and save to test HMR updates.
-        </p>
-        <p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-          {' | '}
-          <a
-            className="App-link"
-            href="https://vitejs.dev/guide/features.html"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Vite Docs
+      <div className="container">
+        {/* Title Section */}
+        <div className="title">
+          <h1 className="title-text">
+            SOMETHING AWESOME<br></br>IS IN THE WORKS
+          </h1>
+          <hr className="line"></hr>
+          <p className="subtitle">
+            Hey F!rosh! We are working hard to give you the best experience, check back soon!
+          </p>
+        </div>
+
+        {/* Registration Info */}
+        <div className="info">
+          <a href="https://www.instagram.com/froshweek/" target="_blank" rel="noreferrer">
+            <img src={InstagramLogo} className="instagram-logo" />
```

Comment:
images should have an alt value, it's good practice (for accessibility). not needed though